lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -6,26 +6,56 @@ class JsonApi < Base
         autoload :PaginationLinks
         autoload :FragmentCache
 
-        def initialize(serializer, options = {})
+        module ApiObjects
+          # Make JSON API top-level jsonapi member opt-in
+          # ref: http://jsonapi.org/format/#document-top-level
+          ActiveModel::Serializer.config.jsonapi_include_toplevel_member = false
+          ActiveModel::Serializer.config.jsonapi_version = '1.0'
+          ActiveModel::Serializer.config.jsonapi_toplevel_meta = {}
+          module JsonApi
```

Comment:
Probably is a latent jsonapi config object here, or at least `JsonApi.config.version` via `class JsonApi < Base; def config; ActiveModel::Serializer.config.jsonapi; end;`